src/dmd/doc.d

```diff
@@ -1815,12 +1814,12 @@ extern (C++) int cmp(const(char)* stringz, const(void)* s, size_t slen)
     return memcmp(stringz, s, slen);
 }
 
-extern (C++) int icmp(const(char)* stringz, const(void)* s, size_t slen)
+private bool icmp(const(char)* stringz, const(char)* s, size_t slen)
```

Comment:
Isn’t the name a bit misleading now when it will only make a comparison.

Reply:
Definitely. Any suggestion ?

Reply:
`equal`, `iequal`, `equali`, `equalCaseInsensitive`.